spec/truffle/pre_initialization_spec.rb

```diff
@@ -9,7 +9,8 @@
 
 require_relative '../ruby/spec_helper'
 
-guard -> { Truffle.native? } do
+# GraalVM does not use the pre-initialized context yet
+guard -> { Truffle.native? and !Truffle::System.get_java_property("org.graalvm.version") } do
```

Comment:
Don't we have a cop for this `and` keyword? :-)

Reply:
Why does it matter that this doesn't run for GraalVM?

Reply:
The specs would fail on GraalVM as GraalVM doesn't build an image with a pre-initialized context yet.